gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java

```diff
@@ -21,10 +24,15 @@
 
   @Override
   public AttributeUpdateOperation convert(final UpdateOperation source) {
-    return AttributeUpdateOperation.newBuilder()
-        .setAttribute(columnIdentifierConverter.convert(source.getAttribute()))
-        .setOperator(operatorConverter.convert(source.getOperator()))
-        .setValue(literalConstantConverter.convert(source.getValue()))
-        .build();
+    final Builder builder =
+        AttributeUpdateOperation.newBuilder()
+            .setAttribute(columnIdentifierConverter.convert(source.getAttribute()))
+            .setOperator(operatorConverter.convert(source.getOperator()));
+
+    if (source.getOperator() != OPERATOR_UNSET) {
```

Comment:
If the operator is unset, why are we setting it on line 30?

Reply:
But why do we need to convert and set the operator? Is unset a special value? Usually if we get an unset value, we don't want to blindly set it in the proto, we want to leave it unset to support presence detection.

Reply:
I think, I understood why we are confused here. 
This is a deliberately created enum value (not the proto generated one)
![image](https://user-images.githubusercontent.com/93120060/206727842-d7ffdf9e-6ec4-4c7a-9516-840132fa1f71.png)

Reply:
Ah - got it, yes I thought this was the unspecified case! thanks for following up.